api/src/apps/boat_tracking/importer.py

```diff
@@ -21,19 +21,20 @@ class ImportBoatTracking:
     For example, the scraped api calls the geo field "Position" while
```

Comment:
Minor detail: You might want to split the comment into two comments: 1 class comment, 1 variable comment.